test/Microsoft.AspNetCore.Rewrite.Tests/IISUrlRewrite/FileParserTests.cs

```diff
@@ -128,6 +128,37 @@ public void RuleParse_ParseMultipleRules()
             AssertUrlRewriteRuleEquality(expected, res);
         }
 
+        [Fact]
+        public void Should_parse_global_rules()
+        {
+            // arrange
+            var xml = @"<rewrite>
+                            <globalRules>
+                                <rule name=""httpsOnly"" patternSyntax=""ECMAScript"" stopProcessing=""true"">
+                                    <match url="".*"" />
+                                    <conditions logicalGrouping=""MatchAll"" trackAllCaptures=""false"">
+                                        <add input=""{HTTPS}"" pattern=""off"" />
+                                    </conditions>
+                                    <action type=""Redirect"" url=""https://{HTTP_HOST}{REQUEST_URI}"" />
+                                </rule>
+                            </globalRules>
+                            <rules>
+                                <rule name=""Rewrite to article.aspx"">
+                                    <match url = ""^article/([0-9]+)/([_0-9a-z-]+)"" />
+                                    <action type=""Rewrite"" url =""article.aspx?id={R:1}&amp;title={R:2}"" />
+                                </rule>
+                            </rules>
+                        </rewrite>";
+
+            // act
+            var rules = new UrlRewriteFileParser().Parse(new StringReader(xml));
+
+            // assert
+            Assert.Equal(2, rules.Count);
```

Comment:
@mikaelm12 frankly, i don't agree. that's an awful lot of setup for no real value. it also distracts from what the test is actually testing. beyond that, it would require that the existing method be updated to be aware of global rules and then all other tests that utilize that method would have to be changed as well. if you still think it should, i would appreciate seeing an example of how this would look.